### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by dianaboiangiu**

### CODE REVIEW

File: gemet/thesaurus/utils.py [hunk 1]
```diff
@@ -177,3 +177,19 @@ def split_text_into_terms(raw_text):
...
+    # returns true if the concept already has a relation with the given
+    #   relation_type ( only for group and broader for Groups)
+    current_relations = Relation.objects.filter(
+        source=concept,
+        property_type__name=relation_type).exclude(
+        status__in=[DELETED_PENDING, DELETED])
+    broader_relation = \
+        relation_type == 'broader' and concept.namespace.heading == 'Groups'
+    group_relation = relation_type == 'group'
+    if (group_relation or broader_relation) and current_relations:
```
Comment (iuliachiriac) [inline gemet/thesaurus/utils.py]: First, if `not (group_relation or broader_relation)`, then return `False`. Then search for current relations.

File: gemet/thesaurus/utils.py [hunk 2]
```diff
@@ -177,3 +177,19 @@ def split_text_into_terms(raw_text):
...
+
+def concept_has_unique_relation(concept, relation_type):
+    # returns true if the concept already has a relation with the given
+    #   relation_type ( only for group and broader for Groups)
+    current_relations = Relation.objects.filter(
+        source=concept,
+        property_type__name=relation_type).exclude(
+        status__in=[DELETED_PENDING, DELETED])
+    broader_relation = \
+        relation_type == 'broader' and concept.namespace.heading == 'Groups'
```
Comment (iuliachiriac) [inline gemet/thesaurus/utils.py]: Avoid hardcoded values: `concept.namespace.heading == Group.NAMESPACE`

File: gemet/thesaurus/utils.py [hunk 3]
```diff
@@ -177,3 +177,19 @@ def split_text_into_terms(raw_text):
...
     return term_list
+
+
+def concept_has_unique_relation(concept, relation_type):
+    # returns true if the concept already has a relation with the given
+    #   relation_type ( only for group and broader for Groups)
+    current_relations = Relation.objects.filter(
+        source=concept,
+        property_type__name=relation_type).exclude(
+        status__in=[DELETED_PENDING, DELETED])
```
Comment (iuliachiriac) [inline gemet/thesaurus/utils.py]: `concept.source_relations.filter(...`

**Comment by dianaboiangiu** [comment]
Changes made.

**Pull request merged**